groceries_app/static/css/base.css

```diff
@@ -1,3 +1,11 @@
+a {
+    text-decoration: none;
+}
+
+a:hover {
+    text-decoration: none;
+}
```

Comment:
If decoration is none, do you need this?

Reply:
For links <a> text decoration in the form of an underline happens automatically.